libV2/tasks/DiskUsage.js

```diff
@@ -129,9 +143,13 @@ class MonitorDiskUsage extends BaseTask {
         return this._program.lock !== undefined;
     }
 
-    _getUsage() {
-        moduleLogger.debug(`calculating disk usage for ${this._path}`);
-        return getFolderSize(this._path);
+    // eslint-disable-next-line class-methods-use-this
+    async _getUsage(path) {
+        moduleLogger.debug(`calculating disk usage for ${path}`);
+        if (!fs.existsSync(path)) {
```

Comment:
Hello! LGTM, I was just curious why the sync method is used in an async function? :slightly_smiling_face:

Reply:
Actually as I read the docs more it seems the callback version is deprecated ([docs](https://nodejs.org/docs/latest-v16.x/api/fs.html#fsexistspath-callback)) and the promise version doesn't exist. `getFolderSize` is calling `stat` under the hood like the docs suggest so I think the sync usage is fine here.